main.css

```diff
@@ -41,10 +41,36 @@ td{
 }
 
 .topbar{
+	display: flex;
+	flex-wrap: nowrap;
```

Comment:
changing to flex-wrap:wrap, makes the flexbox wrap around the sublists as well. making all the flexboxes grow in size.

I think this could easily be covered when we are able to do a mobile ui rework/addition.

Reply:
I'd like to keep mobile at least *usable*. it doesn't have to be pretty, but not being able to click links is kinda a dealbreaker.

Reply:
or you could add overflow:auto to allow scrolling on mobile so you can see the links